bcmserver.c

```diff
@@ -179,9 +180,11 @@ int main(void)
 	saddr.sin_addr.s_addr = htonl(INADDR_ANY);
 	saddr.sin_port = htons(PORT);
 
+	struct timespec r;
 	while(bind(sl,(struct sockaddr*)&saddr, sizeof(saddr)) < 0) {
 		printf(".");fflush(NULL);
-		usleep(100000);
+		struct timespec f = {0, 100 * 1000 * 1000};
```

Comment:
Please move the declaration of the struct to the beginning of the scope, please initialize it using C99 initializers.

```
struct timesepc f = {
	.tv_nsec = 100 * 1000 * 1000,
};
```

Reply:
Done.